acceptance/workspace/jobs/create/test.toml

```diff
@@ -8,3 +9,19 @@ Response.Body = '''
     "job_id": 1111
 }
 '''
+
+[[Repls]]
+Old = "(linux|darwin|windows)"
+New = "[OS]"
+
+[[Repls]]
+Old = " upstream/[A-Za-z0-9.-]+"
+New = ""
+
+[[Repls]]
+Old = " upstream-version/[A-Za-z0-9.-]+"
```

Comment:
I think you can combine this and regex above into one like `upstream(?:-version)?/[A-Za-z0-9.-]+`

Reply:
I'd prefer to keep them separate just because it's easier to grok at a glance.